classes/village/VillageManager.php

```diff
@@ -304,26 +304,12 @@ public static function getResourceHistory(System $system, int $village_id, int $
             $result = $system->db->fetch($result);
             $resource_history[$resource_id]['claimed'] = (int)$result['claimed'];
             // get lost (lol)
-            // can determine resources lost based on difference of produced and collected
-            $resource_history[$resource_id]['lost'] = $resource_history[$resource_id]['produced'] - $resource_history[$resource_id]['collected'];
-            // factor in resources which haven't been collected from regions
-            $result = $system->db->query("SELECT SUM(`resource_count`) as 'quantity' FROM `region_locations` INNER JOIN `regions` on `regions`.`region_id` = `region_locations`.`region_id` WHERE `resource_id` = {$resource_id} AND `village` = {$village_id}");
+            $result = $system->db->query("SELECT COUNT(*) as 'lost' FROM `loot` WHERE `resource_id` = {$resource_id} AND `target_village_id` = {$village_id} AND (`claimed_village_id` != {$village_id} OR `claimed_village_id` IS NULL) AND (`claimed_time` > {$time} OR `claimed_time` IS NULL)");
```

Comment:
Minor, but we could be more clear here

```suggestion
            $result = $system->db->query("SELECT COUNT(*) as 'lost_loot_count' FROM `loot` WHERE `resource_id` = {$resource_id} AND `target_village_id` = {$village_id} AND (`claimed_village_id` != {$village_id} OR `claimed_village_id` IS NULL) AND (`claimed_time` > {$time} OR `claimed_time` IS NULL)");
```

Reply:
Sounds good to me!